.gitignore

```diff
@@ -3,6 +3,7 @@
 
 .coverage/
 rubydeps.*
+*.swp
```

Comment:
:-1: This is something for your personal global gitignore. Currently all crowbar components share the same gitignore. Anyway this ignore is just related to your editor.